images-list

```diff
@@ -110,6 +110,11 @@ quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.16.5
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.16.6
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.17.1
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.17.2
+quay.io/cilium/cilium rancher/cilium-cilium v1.9.4
```

Comment:
This does not exist in dockerhub: https://hub.docker.com/r/rancher/cilium-cilium/

Reply:
@brandond Think we've talked about this before but it would be cool if during CI we did a skopeo inspect on the repo to make sure it exists, and then a 2nd inspect on the tag to make sure it doesn't exist. I will add that as tech debt if you agree. Seems like it would be easy.

Reply:
Pfff my bad, created a new issue: https://github.com/rancherlabs/eio/issues/362

Reply:
Yeah, I have been meaning to add some PR checks. I think we'll need to use the docker API to poke at the repo if its empty, but that's not too difficult.